pcsx2/GS/Renderers/SW/GSRasterizer.cpp

```diff
@@ -1131,7 +1133,9 @@ void GSRasterizer::Flush(const GSVertexSW* vertex, const u32* index, const GSVer
 				int left = e->_pad.I32[1];
 				int top = e->_pad.I32[2];
 
-				DrawScanline(pixels, left, top, *e++);
+				if ((m_scanmsk_value & 2) == 0 || (m_scanmsk_value & 1) == (top & 1))
```

Comment:
Isn't it the opposite? When scanmsk is 2 (so bit 0 will be zero) drawing of even lines is prohibited.  so you want !=